experimental/PropertyDDS/packages/property-shared-tree-interop/jest.config.cjs

```diff
@@ -19,7 +19,7 @@ module.exports = {
 	transform: {
 		"^.+\\.ts$": ["ts-jest", { tsconfig: "src/test/tsconfig.json" }],
 	},
-	testPathIgnorePatterns: ["/node_modules/", "dist", "lib"],
+	testPathIgnorePatterns: ["/node_modules/", "lib"],
```

Comment:
Does dist need to be omitted? Wondering if it's safe to "standardize" this across our configs regardless of project.

Reply:
It's harmless.  I've been leaving dist in 'npm clean'.  We can leave it here as well.